### PR TITLE
add docstring to example: axisartist/demo_floating_axes.py

### PR DESCRIPTION
**Pull request by JunTan**
Adding docstring to matplotlib/examples/axisartist.

### CODE REVIEW

**Comment by NelleV** [comment]
Hi @JunTan 
Thanks or the patch. Overall this looks good! Can you just reformat the docstring such that it is as follow:

"""
title

description
"""

File: examples/axisartist/demo_floating_axes.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
+"""
+Demo of the scatter function and bar function with changing the shape of the plot.
```
Comment (NelleV) [inline examples/axisartist/demo_floating_axes.py]: I'd make the title shorter, and about the features showcases here: something as simple as "Floating axes" would work well here.

Comment (QuLogic) [inline examples/axisartist/demo_floating_axes.py]: Please also wrap all the lines at 79 characters.

**Comment by NelleV** [comment]
That's strange… The pep8 tests fail on lines you did not modify.
@tacaswell is that normal?

**Comment by NelleV** [comment]
@tacaswell never mind… I confused lines and colums in the pep8 error message.

**Comment by QuLogic** [comment]
Backported to v2.x via 35aa62dec4fb82a12af38a84d0f1ea397539ea4c.

**Pull request merged**